src/pages/questionPage.js

```diff
@@ -23,13 +23,25 @@ export const initQuestionPage = (userInterface) => {
     .addEventListener('click', nextQuestion)
 };
 // check answers if correct or not 
+let isSelectedOneAnswer= false;
 const getAnswer = (e)=> {
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+if(!isSelectedOneAnswer){
   const answer = e.target.innerText;
   currentQuestion.selected=answer.charAt(0)
-  currentQuestion.selected===currentQuestion.correct ? console.log('selected correct answer') : console.log('selected wrong answer')
+  showAnswerIsCorrect(currentQuestion.selected, currentQuestion.correct,e.target)
+  isSelectedOneAnswer=true;
 }
+}
+// Show user if user selection correct or not
+const showAnswerIsCorrect =(selectedAnswer,correctAnswer,target)=> {
```

Comment:
the `=` before `(selectedAnswer` is a typo?

Reply:
It is for defining arrow function const arrowFunc = () => { }

Reply:
haha, sorry, I normally leave a space between them, but it's good as it is